static/src/js/containers/FooterContainer/FooterContainer.js

```diff
@@ -38,50 +118,19 @@ export class FooterContainer extends Component {
             <span className="footer__ver-pill">
               {
                 edscEnv !== 'prod' && (
-                  <span className="footer__env">
+                  <span className={pillClassNames}>
                     {edscEnv.toUpperCase()}
                   </span>
                 )
               }
-              {`v${edscVersion}`}
-            </span>
-            {(searchTimeVisible && loadTime !== 0) && (
-              <span className="footer__info-bit footer__info-bit--emph">
-                {`Search Time: ${loadTimeInSeconds}s`}
-              </span>
-            ) }
-            <span className="footer__info-bit footer__info-bit--emph">
-              NASA Official: Stephen Berrick
-            </span>
-            <span className="footer__info-bit">
-              <a
-                className="footer__info-link"
-                href="http://www.nasa.gov/FOIA/index.html"
-              >
-                FOIA
-              </a>
-            </span>
-            <span className="footer__info-bit">
-              <a
-                className="footer__info-link"
-                href="http://www.nasa.gov/about/highlights/HP_Privacy.html"
-              >
-                NASA Privacy Policy
-              </a>
-            </span>
-            <span className="footer__info-bit">
-              <a className="footer__info-link" href="http://www.usa.gov">USA.gov</a>
+              {version()}
             </span>
+            {searchTime()}
```

Comment:
Should we make a `FooterLink` component fr each of these.. footer links?

Reply:
I don't hate the idea 🙂